### ISSUE TITLE
data-default-value option seems not to work properly

### ISSUE THREAD

**Issue by alexdruk** [benkeen/country-region-selector#37]
The following code still show first region from the list (Alsace), not default value:
```
		<div class="form-group">
          <label for="country">Country</label>
           		  <select name="country" class="form-control crs-country" data-region-id="one" data-value="shortcode" data-default-value="FR" data-show-default-option="false">
 		   		  </select>
        </div>
		<div class="form-group">
          <label for="state">State or region</label>
           		  <select name="state" class="form-control" id="one" data-default-value="Franche-Comté"  data-show-default-option="false">
		   		  </select>
        </div>

```

**Comment by benkeen** [comment]
Good catch, thanks @alexdruk. I'll take a look.

**Comment by benkeen** [comment]
This has been fixed in 0.3.4 - just released.
